src/app/qgshandlebadlayers.cpp

```diff
@@ -509,3 +479,161 @@ int QgsHandleBadLayers::layerCount()
 {
   return mLayerList->rowCount();
 }
+
+QString QgsHandleBadLayers::findFile( const QString &fileName, const QString &basePath, int maxDepth, const int driveMargin )
```

Comment:
```suggestion
QString QgsHandleBadLayers::findFile( const QString &fileName, const QString &basePath, int maxDepth, int driveMargin )
```

... or is there a reason for this const? If not I'd remove it to be inline with the rest of our API.

Reply:
to prevent change during execution, mostly a safety measure. Should be removed now.